pkg/test/framework/testcontext.go

```diff
@@ -215,8 +214,7 @@ func (c *testContext) AllClusters() cluster.Clusters {
 }
 
 func (c *testContext) CreateDirectory(name string) (string, error) {
-	dir := filepath.Join(c.workDir, name)
-	err := os.Mkdir(dir, os.ModePerm)
+	dir, err := os.MkdirTemp(c.workDir, name)
```

Comment:
avoid issue here we run test twice and use the same thing. code copied from suite context